pybio/spec/utils.py

```diff
@@ -67,17 +73,12 @@ def transform_list(self, node: list) -> list:
         return [self.transform(subnode) for subnode in node]
 
 
-def _resolve_import(importable: nodes.ImportableSource):
-    if isinstance(importable, nodes.ImportableModule):
-        module = importlib.import_module(importable.module_name)
-        return getattr(module, importable.callable_name)
-    elif isinstance(importable, nodes.ImportablePath):
-        importlib_spec = importlib.util.spec_from_file_location(f"user_imports.{uuid.uuid4().hex}", importable.filepath)
-        dep = importlib.util.module_from_spec(importlib_spec)
-        importlib_spec.loader.exec_module(dep)
-        return getattr(dep, importable.callable_name)
+@dataclasses.dataclass
+class ImportedSource:
+    callable_: Callable
```

Comment:
Maybe factory is a better name? It communicates that this callable used to create objects.

Reply:
agreed